tests/services/test_service.py

```diff
@@ -136,3 +136,18 @@ def test_missing_or_invalid_type(lang_data2, service, identity):
         lang_data2['props']['newkey'] = v
         pytest.raises(
             ValidationError, service.create, identity, lang_data2)
+
+
+def test_read_all_no_cache(lang_type, lang_data_many, service, identity):
```

Comment:
For future reference, maybe to be removed in a next PR. If `lang_data_many` uses `lang_type` there is no need to apply it here :)